modules/salt/pages/salt-states.adoc

```diff
@@ -0,0 +1,210 @@
+[[salt-states]]
+= Salt States
+
+Salt is capable of applying states by matching minions with relevant state data.
+This data comes from {productname} in the form of package and custom states.
+
+
+State data comes from {productname} in the form of package and custom states and targets minions at three specific levels of hierarchy.
+The state hierarchy is defined by the following order or priority: individual minions have priority on packages and custom states over groups; next a group has priority over the organization.
+
+* Minion Level
++
+menu:Systems[Specific Minion > States]
+* Group Level
++
+menu:Systems[System Groups]
+* Organization Level
++
+menu:Systems[Manage System Types: > My Organization]
+
+
+For example:
+
+* Org1 requires that vim version 1 is installed
+* Group1 requires that vim version 2 is installed
+* Group2 requires any version installed
+
+
+This would lead to the following order of hierarchy:
+
+* Minion1 part of [Org1, Group1] wants vim removed, vim is removed (Minion Level)
+* Minion2 part of [Org1, Group1] wants vim version 2 gets version 2 (Group Level)
+* Minion3 part of [Org1, Group1] wants any version, gets version 2 (Org Level)
+* Minion4 part of[Org1, Group2] wants any version, gets vim version 1 (Org Level)
+
+
+[[bp.chapt.config.mgr.salt.states.storage.locations]]
+== Salt States Storage Locations
+
+
+The {productname} salt-master reads its state data from three file root locations.
+
+The directory [path]``/usr/share/susemanager/salt``
+ is used by {productname} and comes from the susemanager-sls.
+It is shipped and updated together with {productname} and includes certificate setup and common state logic to be applied to packages and channels.
+
+The directory [path]``/srv/susemanager/salt`` is generated by {productname} and based on assigned channels and packages for minions, groups and organizations.
+This file will be overwritten and regenerated.
+This could be thought of as the {productname} database translated into salt directives.
+
+The third directory [path]``/srv/salt`` is for custom state data, modules etc.
+{productname} does not operate within or utilize this directory.
+However the state data placed here affects the Highstate of minions and is merged with the total state result generated by {productname}.
+
+[[bp.chapt.config.mgr.susemgr.states]]
+== {productname} States
+
+
+All sls files created by users will be saved to disk on the salt-master server.
```

Comment:
```suggestion
All user created SLS files will be saved to disk on the salt-master server.
```
Do we actually use salt-master?  Or Salt master.
I vote to cap SLS as you usually write PNG files, etc.